flent/runners.py

```diff
@@ -680,8 +680,14 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            ss_args = {}
+            for k, v in args:
+                if k in ["host", "interval", "length", "target", "ip_version"]:
+                    ss_args[k] = v
```

Comment:
Hmm, looping over everything in args is somewhat inefficient. Also, the parameters are not called the same, so you'll need to map them. Using a dict literal is probably the easiest. Something like:

```
ss_args = {'host': self.remote_host or 'localhost', 
           'interval': args['interval'],
           'length': args['length'],
           'target': args['host'],
           'ip_version': args['ip_version']}
````